example/BlazorDB.Example/wwwroot/index.html

```diff
@@ -1,26 +1,26 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-    <title>BlazorDB.Example</title>
-    <base href="/" />
-    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
-    <link href="css/app.css" rel="stylesheet" />
-</head>
-
-<body>
-    <app>Loading...</app>
-
-    <div id="blazor-error-ui">
-        An unhandled error has occurred.
-        <a href="" class="reload">Reload</a>
-        <a class="dismiss">🗙</a>
-    </div>
-    <script src="_framework/blazor.webassembly.js"></script>
-    <script src="_content/BlazorDB/dexie.min.js"></script>
-    <script src="_content/BlazorDB/blazorDB.js"></script>
-</body>
-
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>BlazorDB.Example</title>
+    <base href="/" />
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/app.css" rel="stylesheet" />
+</head>
+
+<body>
+    <app>Loading...</app>
+
+    <div id="blazor-error-ui">
+        An unhandled error has occurred.
+        <a href="" class="reload">Reload</a>
+        <a class="dismiss">🗙</a>
+    </div>
+    <script src="_framework/blazor.webassembly.js"></script>
+    <script src="_content/BlazorIndexedDB/dexie.min.js"></script>
+    <script src="_content/BlazorIndexedDB/blazorDB.js"></script>
```

Comment:
I don't think this folder should change, but I'll pull locally to test

Reply:
It was not working with the previous value on my computer, I was thinking it was an unseen regression, so I changed it to make it works on my side. So if it works from your side, there is something I have to understand...

Reply:
> It was not working with the previous value on my computer, I was thinking it was an unseen regression, so I changed it to make it works on my side. So if it works from your side, there is something I have to understand...

For me it only works with your change too. It looks like "BlazorIndexedDB" is the correct path